### ISSUE TITLE
Unyson Plugin Removed from WordPress.org

### ISSUE THREAD

**Issue by manish-webosmotic** [ThemeFuse/Unyson#4391]
Hello,  

I wanted to bring to your attention that the **Unyson plugin** has been removed from WordPress.org. This plugin is actively being used on one of our client's websites.  

Could you please confirm:  
1. Whether it's safe to continue using the plugin in its current state?  
2. Will we still receive regular updates or security patches from the developers?  

If this plugin is no longer supported or poses any risks, we may need to explore alternatives or start planning for a replacement. Please advise on the next steps.  

Thank you!  

Best regards,  
Manish

**Comment by ibbosco** [comment]
Which plug-in can I use alternatively? my site practically no longer works because of this plugin

**Comment by Tipsyvision** [comment]
Hold on a second, does this mean that unyson is not getting any updates anymore????

I AM NOT PLANNING TO USE ELEMENTOR AFTER 3 YEARS OF UNYSON IMAGINE THE HUGE TRANSFER AND PROBLEMS TO SWITCHING TO A OTHER THEME PLUGIN!!!

**Comment by Niloys7** [comment]
@Tipsyvision sad, but it's true. This plugin is abandoned, and you need to find an alternative

**Comment by VishwaS-22** [comment]
Can you suggest me some plugin which does the same work like unyson?
I need it for my teammate.

**Comment by Niloys7** [comment]
@VishwaS-22 It depends on your needs; for creating meta boxes and options, I have switched to the Codestar Framework.

**Comment by VishwaS-22** [comment]
I spoke with my teammate, he is saying like, he is using unyson plugin for installing the demo content.
In that case, can you able to guide or give any suggestions?

**Comment by Niloys7** [comment]
Yes, the Unyson Demo import extension was great. as an alternative I am now using this one https://wordpress.org/plugins/one-click-demo-import/

**Comment by VishwaS-22** [comment]
Oh thanks for the suggestion, I'll share this with my teammate. Thanks!
